hasjob/models/jobpost.py

```diff
@@ -587,6 +588,17 @@ def viewstats_helper(jobpost_id, interval, limit, daybatch=False):
     cviewed = batches * [0]
     copened = batches * [0]
     capplied = batches * [0]
+    cbuckets = batches * ['']
+
+    now_local = datetime.now()
```

Comment:
Use [Baseframe's localize_timezone](https://github.com/hasgeek/baseframe/blob/f053e17381691b8ae01ad81e694fe5788852adf3/baseframe/__init__.py#L236-L243) for this:

```python
from baseframe import localize_timezone
usertime = localize_timezone(servertime)
```